.github/workflows/wheels.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         - [macos-14, macosx, arm64, openblas, "12.3"]
         - [macos-14, macosx, arm64, accelerate, "14.0"]
         - [windows-2019, win, AMD64, "", ""]
-        python: [["cp310", "3.10"], ["cp311", "3.11"], ["cp312", "3.12"], ["cp313", "3.13"]]
+        python: [["cp310", "3.10"], ["cp311", "3.11"], ["cp312", "3.12"], ["cp313", "3.13"], ["cp313t", "3.13"]]
```

Comment:
```suggestion
        python: [["cp310", "3.10"], ["cp311", "3.11"], ["cp312", "3.12"], ["cp313", "3.13"], ["cp313t", "3.13"]]
        exclude:          
          - buildplat: [windows-2019, win, AMD64, "", ""]
            python: ["cp313t", "3.13"]
```

Reply:
My suggestion is preferred because it doesn't generate the matrix entry for windows in the first place.

Reply:
@andfoy, are you going to include this suggestion?

Reply:
See https://github.com/scipy/scipy/pull/21793#issuecomment-2455944671

Reply:
The comment was right but no longer applies indeed since Windows wheels build now, so resolving this comment.